build.config.js

```diff
@@ -69,7 +69,8 @@ module.exports = {
       'src/styles/mixins/**/*.scss',
       'src/styles/common/**/*.scss',
       'src/styles/modules/**/*.scss',
-      'src/app/**/*.scss'
+      'src/app/**/*.scss',
+      '!src/app/**/*.component.scss'
```

Comment:
This is apart of the build process; though I'm not sure about the exclusion of the component's `scss`. What's this doing?